module/zfs/dmu_send.c

```diff
@@ -1076,7 +1064,8 @@ do_dump(dmu_send_cookie_t *dscp, struct send_range *range)
 			} else {
 				psize = BP_GET_PSIZE(bp);
 			}
-			err = dump_write(dscp, srdp->obj_type, range->object,
+			err = dmu_dump_write(dscp, srdp->obj_type,
+			    range->object,
```

Comment:
`offset, srdp->datablksz, psize, bp,` can all be added to this line and still be under the limit.